src/c/custom_codes.h

```diff
@@ -8,7 +8,15 @@
 #define INV_WINDOW_VALUE      4
 #define DIALOGUE_WINDOW_VALUE 2
 
+#define BATTLE_USER_DATA_BASE 0x42
+#define BATTLE_USER_INFO_BASE 0x5C
+#define BATTLE_USER_VAL_BASE  0x92
```

Comment:
There's enough fields and pointer math going on with these numbers that we should probably make an actual struct for Battler data instead of using a byte*. It seems like the logic for `load_The_user_target` closely follows FIX_USER_NAME and FIX_TARGET_NAME in EarthBound, so we've already got decent names for the fields used here (these are just suggestions based on ebsrc and other discussions):

* BATTLE_USER_INFO_BASE: `ally_or_enemy` (or `is_enemy`?)
* BATTLE_USER_INFO_BASE + 1: `npc_id`
* BATTLE_USER_DATA_BASE: `id`
* BATTLE_USER_INFO_BASE - 3: `letter`
* BATTLE_USER_VAL_BASE: `enemy_id`